source/win-main/MainSidebar.vue

```diff
@@ -279,17 +279,20 @@ export default defineComponent({
     citationKeys: function (): string[] {
       return this.$store.state.citationKeys
     },
+    bibliography: function (): any {
+      return this.$store.state.bibliography
+    },
     referenceHTML: function (): string {
-      if (this.bibContents === undefined || this.bibContents[1].length === 0) {
```

Comment:
`bibContents` is then not used anymore in this file, right?

Reply:
Good catch! Will update …